pkg/release-controller/types.go

```diff
@@ -471,6 +471,8 @@ const (
 	// is considered an input image stream for creating releases.
 	ReleaseAnnotationConfig = "release.openshift.io/config"
 
+	ReleaseAnnotationInconsistency = "release.openshift.io/inconsistency"
```

Comment:
Please add a comment about the annotation.  What's it for used for and where.

Reply:
Done.